<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateWindow</class>
 <widget class="QMainWindow" name="TemplateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>樣板</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>基本資料 (必填)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="3" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>保險類別</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>卡片號碼</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>國籍</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>病歷號碼</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="comboBox_ins_type"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>身份證號</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>出生日期</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>病患姓名</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEdit_birthday"/>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_id"/>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>初診日期</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_name"/>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_patient_key">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="3" colspan="2">
              <widget class="QLineEdit" name="lineEdit_card_no"/>
             </item>
             <item row="2" column="3" colspan="2">
              <widget class="QComboBox" name="comboBox_nationality"/>
             </item>
             <item row="3" column="3" colspan="2">
              <widget class="QLineEdit" name="lineEdit_init_date"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>通訊資料</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>聯絡電話</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_telephone"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>行動電話</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>電子郵件</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>聯絡地址</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="3">
              <widget class="QLineEdit" name="lineEdit_email"/>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="lineEdit_cellphone"/>
             </item>
             <item row="2" column="1" colspan="3">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLineEdit" name="lineEdit_address"/>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>其他資料</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="3">
              <widget class="QComboBox" name="comboBox_education"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>婚姻狀況</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>教育程度</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_occupation"/>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox_marriage"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>職業類別</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>優待類別</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QComboBox" name="comboBox_discount">
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>聯絡人</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_family"/>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>聯絡電話</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QLineEdit" name="lineEdit_family_telephone"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>說明</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit_description"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="rightMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>過敏</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTextEdit" name="textEdit_allergy"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>病史</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTextEdit" name="textEdit_history"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>備註</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QTextEdit" name="textEdit_remark"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_save"/>
   <addaction name="separator"/>
   <addaction name="action_close"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1115</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>檔案</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_save">
   <property name="icon">
    <iconset>
     <normaloff>../icons/document-save.svg</normaloff>../icons/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>資料存檔</string>
   </property>
   <property name="toolTip">
    <string>存檔</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset>
     <normaloff>../icons/application-exit.svg</normaloff>../icons/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>關閉視窗</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit_patient_key</tabstop>
  <tabstop>lineEdit_card_no</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_birthday</tabstop>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>comboBox_nationality</tabstop>
  <tabstop>comboBox_ins_type</tabstop>
  <tabstop>lineEdit_init_date</tabstop>
  <tabstop>lineEdit_telephone</tabstop>
  <tabstop>lineEdit_cellphone</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>lineEdit_address</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>comboBox_marriage</tabstop>
  <tabstop>comboBox_education</tabstop>
  <tabstop>comboBox_occupation</tabstop>
  <tabstop>comboBox_discount</tabstop>
  <tabstop>lineEdit_family</tabstop>
  <tabstop>lineEdit_family_telephone</tabstop>
  <tabstop>textEdit_description</tabstop>
  <tabstop>textEdit_allergy</tabstop>
  <tabstop>textEdit_history</tabstop>
  <tabstop>textEdit_remark</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
